pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -155,10 +155,12 @@ func (r *ReconcileService) Reconcile(request reconcile.Request) (reconcile.Resul
 	configurationReconciler := knative.NewConfigurationReconciler(r.Client)
 	routeReconciler := knative.NewRouteReconciler(r.Client)
 
+	// Initialize conditions
+	kfsvc.Status.InitializeConditions()
```

Comment:
What exactly is this doing? Setting them all to unknown? Why is this necessary to setup?